web/src/engine/websites/MangaLib_e2e.ts

```diff
@@ -1,24 +1,47 @@
 ﻿import { TestFixture } from '../../../test/WebsitesFixture';
 
-const config = {
+const configWithBranches = {
     plugin: {
         id: 'mangalib',
         title: 'MangaLib'
     },
     container: {
-        url: 'https://mangalib.org/toukyou-revengers',
-        id: '/toukyou-revengers',
-        title: 'Токийские мстители'
+        url: 'https://mangalib.org/ru/manga/7965--chainsaw-man',
+        id: '7965--chainsaw-man',
+        title: 'Человек-бензопила'
     },
     child: {
-        id: '/toukyou-revengers/v1/c1',
-        title: '1 - Reborn'
+        id: JSON.stringify({ branch_id: '4667', number: '1', volume: '1' }),
+        title: 'Том 1 Глава 1 - Пёс и бензопила [Nippa Team]'
     },
     entry: {
-        index: 1,
-        size: 100_108,
-        type: 'image/jpeg'
+        index: 4,
+        size: 314_272,
+        type: 'image/webp'
     }
 };
 
-new TestFixture(config).AssertWebsite();
\ No newline at end of file
+new TestFixture(configWithBranches).AssertWebsite();
+
+const configWithoutBranches = {
```

Comment:
May directly embed this config in `TestFixture` constructor to avoid intermediate variable.
When removing the variable I would suggest to add a comment describing the test case.

Reply:
Yeah, it has not being updated since we started to changes tests.